internal/webhook/manager.go

```diff
@@ -27,8 +33,8 @@ import (
 // - serving.kserve.io/v1beta1: inferenceservices
 
 const (
-	WebhookServiceName                 = "opendatahub-operator-webhook-service"
-	ValidatingWebhookConfigurationName = "opendatahub-operator-validating-webhook-configuration"
+	WebhookServiceName                 = "opendatahub-operator-controller-manager-service"
+	ValidatingWebhookConfigurationName = "opendatahub-operator-validating-webhook-configuration-managed"
 	AdmissionReviewVersion             = "v1"
```

Comment:
The code now consistently refers to the service name `opendatahub-operator-controller-manager-service`, as both OLM installations and `make deploy` now use the same service name.